tools/cast_ipython.rc

```diff
@@ -0,0 +1,16 @@
+# This file contains ipython configuration variables to be used for generating
+# asciinema demos to guarantee consistent appearance.
+
+# make a fake temporary home dir and go into it
+SCREENCAST_HOME=~/demo
```

Comment:
use
```bash
SCREENCAST_HOME=${SCREENCAST_HOME:-/demo}
```
as we have in cast_bash.rc?  may be even just source `cast_bash.rc` to get also git configured etc?